components/InfoCard.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import Image from "next/image";
+import { HeartIcon } from "@heroicons/react/outline";
+import { StarIcon } from "@heroicons/react/solid";
+import { useRouter } from "next/router";
+import ReactPlayer from "react-player";
+function InfoCard({
+  containsLockedLessons,
```

Comment:
Too many arguments in function, avoid it. And some of them are unusable.

Reply:
Fixed)